apple-pay-react-ts-checkout-netlify/client/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>React TS Checkout</title>
```

Comment:
Do you want to change the title to ApplePay Checkout?

Reply:
Change the title now.